src/content/docs/network-performance-monitoring/monitoring-network-data/visualize-network-data.mdx

```diff
@@ -32,57 +34,70 @@ Once you've set up these visualizations, you'll be able to start exploring the n
 
-- Select the account you want to add the dashboard to.
-- Edit the dashboard name, if you want to.
+- Select the account where you want to add the dashboard.
+- Optional: Edit the dashboard name.
 
 4. Click **Import dashboard**.
 
 <Callout variant="tip">
-  If you want to read more about custom visualizations in New Relic One, see the
-  [introduction to custom
-  visualizations](https://developer.newrelic.com/explore-docs/custom-viz/).
+  If you want to read more about custom visualizations in New Relic One, see the [introduction to custom visualizations](https://developer.newrelic.com/explore-docs/custom-viz/).
 </Callout>
 
 ## Start exploring your network performance data [#explore-network-performance-data]
 
-Once you've finished setting up these dashboards, you can start seeing your network data by going to:
-
-- **[one.newrelic.com](https://one.newrelic.com)** > **Dashboards**:
-
-  - For usage trend analysis, you'll see the following dashboard:
-    ![Data Ingest and Cardinality](./images/dashboard_Data_Ingest_and_Cardinality.png 'Data Ingest and Cardinality')
-    <figcaption>
-      Dashboard for analyzing network telemetry usage trends in New Relic One.
-    </figcaption>
-  - For SNMP data, you'll see the following dashboard:
-    ![SNMP Routers and Switches](./images/dashboard_Routers_and_Switches.png 'SNMP Routers and Switches')
-    <figcaption>
-      Dashboard for SNMP data from Router and Switch entities in New Relic One.
-    </figcaption>
-  - For Network flow data, you'll see the following dashboard:
-    ![Network flow data monitoring](./images/flow_data_dashboard.png 'Network flow data monitoring')
-    <figcaption>
-      Dashboard for Network flow data monitoring in New Relic One.
-    </figcaption>
-
-- **[one.newrelic.com](https://one.newrelic.com)** > **Explorer**:
-  - Entities are listed under the **Network** category in the left-hand side menu.
-  - Open a specific device to see the details and trend of its performance.
-  - On the top right-hand side, click **Lookout** to see anomalies.
-    ![Lookout view](./images/snmp_dashboard.png 'Lookout view')
-    <figcaption>
-      Lookout view for Network Performance Monitoring in New Relic One.
-    </figcaption>
+Once you've finished setting up these dashboards, you can start seeing your network data by going to **Dashboards** or **Explorer** in New Relic One:
+
+From **[one.newrelic.com](https://one.newrelic.com)** > **Dashboards**:
+
+For usage trend analysis, you'll see the following dashboard:
+
+![Data Ingest and Cardinality](./images/dashboard_Data_Ingest_and_Cardinality.png 'Data Ingest and Cardinality')
+
+<figcaption>
+  Dashboard for analyzing network telemetry usage trends in New Relic One.
+</figcaption>
+
+For SNMP data, you'll see the following dashboard:
+
+![SNMP Routers and Switches](./images/dashboard_Routers_and_Switches.png 'SNMP Routers and Switches')
+
+<figcaption>
+  Dashboard for SNMP data from Router and Switch entities in New Relic One.
```

Comment:
Router and Switch or router and switch?

Reply:
hmm... this is how it was in original text. I'll l/c it. Good catch!